.github/workflows/ci-pipeline.yml

```diff
@@ -0,0 +1,30 @@
+name: CI Automate
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches: [develop, '**']
```

Comment:
I believe the pull requests that needs the run is only **Develop**, unless we want this for every single branch, then we can use the stars only

Reply:
Yes, I think we would need a build to pass on every branch as they might be some fixes PR that do not go directly to the develop branch